plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -81,14 +123,88 @@ class PodDebugTabView extends React.Component {
     return <DescriptionList hash={LastVersionChangeValueMapping} />;
   }
 
+  getRecentOfferSummary() {
+    const queue = this.props.pod.getQueue();
+    let introText = null;
+    let mainContent = null;
+    let offerCount = null;
+
+    if (queue == null || queue.declinedOffers.summary == null) {
+      introText = 'Offers will appear here when your service is deploying or waiting for resources.';
+    } else {
+      const {declinedOffers: {summary}} = queue;
+      const {roles: {offers = 0}} = summary;
+      const docsURL = MetadataStore.buildDocsURI(
+        '/overview/concepts/#mesos-resource-offer'
+      );
+
+      introText = (
+        <span>
+          When you attempt to deploy a service, DC/OS waits for offers to match the resources your service requires. If the offer does not satisfy the requirement, it is declined and DC/OS retries. <a href={docsURL} target="_blank">Learn more</a>.
```

Comment:
I guess we're deliberately abstracting Marathon and Mesos away here? Feels kinda weird to be talking about DC/OS itself doing either of these things.